src/ig/IGExporter.ts

```diff
@@ -155,6 +155,18 @@ export class IGExporter {
         exampleBoolean: false
       });
     });
+    this.pkg.instances.forEach(instance => {
+      const instanceFileName = `${instance.resourceType}-${instance.id ?? instance.instanceName}`;
+      const instancePath = path.join(igPath, 'input', 'instances', `${instanceFileName}.json`);
+      outputJSONSync(instancePath, instance, { spaces: 2 });
```

Comment:
Just to double-check: this is supposed to include the `instanceName` field, which would get removed if only the resources were being generated in `app.ts` line 81.

Reply:
That's a great catch! The `instanceName` field should be removed here as well. I'll fix that.